src/modules/edc-demo/components/contract-definition-viewer/contract-definition-viewer.component.html

```diff
@@ -35,14 +33,43 @@
 
     <!-- Paginator -->
     <mat-paginator
+      *ngIf="contractDefinitionList.isReady"
       hidePageSize="true"
       pageSize="10"
-      [length]="contractDefinitions?.length"></mat-paginator>
+      [length]="
+        contractDefinitionList.data.filteredContractDefinitions.length
+      "></mat-paginator>
   </div>
 
-  <div fxLayout="row wrap" fxLayoutAlign="start start">
+  <div class="flex flex-row uppercase justify-center">
```

Comment:
The uppercase class here is right now making empty messages uppercase.

It should be removed here and empty state / error state should specify such styling.